gateway-ha/gateway-ha-config.yml

```diff
@@ -1,25 +1,30 @@
 routingRules:
-  rulesEngineEnabled: False
-  # rulesConfigPath: "src/main/resources/rules/routing_rules.yml"
+  rulesEngineEnabled: True
```

Comment:
Again something that can be put into some internal repo

Reply:
already this is a part of uptycs-kubernetes repo. This just for local setup